paramak/shape.py

```diff
@@ -1229,6 +1210,6 @@ def export_graveyard(
         """
 
         self.make_graveyard(graveyard_offset=graveyard_offset)
-        self.graveyard.export_stp(Path(filename))
+        new_filename = self.graveyard.export_stp(Path(filename))
 
-        return filename
+        return new_filename
```

Comment:
these added lines aren't tested

Reply:
thanks for spotting this, I can add tests for that